platform/platform_filesystem.c

```diff
@@ -440,7 +400,7 @@ bool plCopyFile(const char *path, const char *dest) {
     return false;
 }
 
-size_t plGetFileSize(const char *path) {
+size_t plGetLocalFileSize(const char *path) {
```

Comment:
Probably worth considering killing this completely in favour of plGetFileSize. What do you think @solemnwarning ?